site/content/docs/5.1/getting-started/introduction.md

```diff
@@ -68,49 +103,13 @@ Curious which components explicitly require our JavaScript and Popper? Click the
 {{< /markdown >}}
```

Comment:
we have missed popovers (and [tabs](https://deploy-preview-36204--twbs-bootstrap.netlify.app/docs/5.1/components/navs-tabs/#javascript-behavior)) here :open_mouth: